pkg/app/provision.go

```diff
@@ -18,6 +24,26 @@ type ProvisionFlags struct {
 	Inventory string
 }
 
+type ValidateFlags struct {
+	RootFlags
+
+	ServiceSubnet string
+	PodSubnet     string
+	APIServerPort int
+}
+
+func Validate(inventory string, flags ValidateFlags) error {
```

Comment:
This file is named `provision.go`, lets create another file `validate.go` in the same package that has the `validate` specific code.